src/core/core.ts

```diff
@@ -413,11 +413,11 @@ class Zoomist {
   }
 
   #useDblTouch(e: TouchEvent) {
-    e.preventDefault();
     if (e.touches.length !== 1) {
       return;
     }
     if (Date.now() - this.data.dblTouchData.lastTouchTime < 300) {
+      e.preventDefault();
```

Comment:
On mobile safari double touch works great. But on google chrome, with device toolbar on, it neither toggle fullscreen or zoom. Also. zoom does not work in full screen on chrome

Reply:
I tested in on my mobile phone and it works on chrome. There is an issue with google dev tools